Makefile

```diff
@@ -47,7 +47,7 @@ configure:  # does any pre-requisite installs
 	@#pip install poetry==1.1.9  # this version is known to work. -kmp 11-Mar-2021
 	# Pin to version 1.1.15 for now to avoid this error:
 	#   Because encoded depends on wheel (>=0.29.0) which doesn't match any versions, version solving failed.
-	pip install wheel==0.37.1
+	pip install wheel==0.40.0
```

Comment:
You mean bumping wheel to 0.40.0? Just thought something like that should be the latest?

Reply:
For what it's worth, CGAP is using `wheel 40.0`, even though it doesn't say so:
```
(cgenv) kentpitman@Kents-MacBook-Pro ~/py/cgap-portal$ grep wheel pyproject.toml
wheel = ">=0.29.0"
(cgenv) kentpitman@Kents-MacBook-Pro ~/py/cgap-portal$ grep wheel- poetry.
grep: poetry.: No such file or directory
(cgenv) kentpitman@Kents-MacBook-Pro ~/py/cgap-portal$ grep wheel- poetry.lock
    {file = "wheel-0.40.0-py3-none-any.whl", hash = "sha256:d236b20e7cb522daf2390fa84c55eea81c5c30190f90f29ae2ca1ad8355bf247"},
    {file = "wheel-0.40.0.tar.gz", hash = "sha256:cd1196f3faee2b31968d626e1731c94f99cbdb67cf5a46e4f5656cbee7738873"},
```
There is some _very_ slight motivation for the lower level repos to use something like `>=0.29.0` because they want to be adaptable to what other repos built on them want to use, but even there we probably want to periodically bump the requirement to the minimum of what we know is in use because such old versions slow down poetry solving.